test/unit/git_test.rb

```diff
@@ -10,20 +10,23 @@ def test_git_url_https
     GitHelpers.expects(:which_github_host).returns("www.testmagic.com")
     GitHelpers.expects(:git_info_for_stack).returns({:stack => {:repository => 'drills', :user => 'construction'}})
     GitHelpers.expects(:git_info_for_stack).returns({:stack => {:repository => 'drills', :user => 'construction'}})
+    GitHelpers.expects(:git_info_for_stack).returns({:stack => {:repository => 'drills', :user => 'construction'}})
     assert_equal('https://www.testmagic.com/construction/drills.git', GitHelpers.git_url(:stack, 'https', false))
   end
 
   def test_git_url_default
     GitHelpers.expects(:which_github_host).returns("www.testmagic.com")
     GitHelpers.expects(:git_info_for_stack).returns({:stack => {:repository => 'drills', :user => 'construction'}})
     GitHelpers.expects(:git_info_for_stack).returns({:stack => {:repository => 'drills', :user => 'construction'}})
+    GitHelpers.expects(:git_info_for_stack).returns({:stack => {:repository => 'drills', :user => 'construction'}})
```

Comment:
If you delete the extra lines and add .at_least_once to the end it should fix this